start.sh

```diff
@@ -1,7 +1,3 @@
 #!/bin/sh
 
-REDIS_PASSWORD="${REDIS_PASSWORD:-redis}"
-
-echo "requirepass \"$REDIS_PASSWORD\"" >> /usr/local/etc/redis/redis.conf
```

Comment:
Why do we want to remove password auth?

Reply:
It conflicts with the balena registry implementation, which does not support Redis authentication. It's unnecessary in our production system as well.

Reply:
@Page- To my knowledge this container is only used in product-os, do you know of anywhere else it is used?

Reply:
I thought it was used in BoB but I've just checked and apparently it uses `redis:alpine` directly. Do you know what this repo does specially that product-os needs? (and depending on what it does maybe it should be moved to the product-os repo)

Reply:
@Page- Digging around, my best guess is that this was meant to be used as part of a contract based deployment. Let me have a think and look at the conf file, maybe we can just remove this and use `redis:alpine` directly, as IMO this is the best approach

Reply:
In general, I want to homogenise as much as possible between balena and jellyfish

Reply:
Yup, that's why I brought it up. As a second best having it in product-os only raises flags that it's product-os specific rather than shared

Reply:
I'll move it now, since it will need to happen either way